src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -76,7 +120,41 @@ public void storeMessage(Message message) {
         e.printStackTrace();
       }
     }
+    return messages;
+    */
+  }
 
+  /*
+   * Gets messages posted by all users.
+   * 
+   * @return a list of messages posted by all users, or empty list if user has never posted a 
+   *      message. List is sorted by time descending. 
+   */
+  public List<Message> getAllMessages() {
+    return messages_list("");
+    /*
```

Comment:
Why is all of this code commented out? Is there a reason for this? It is usually bad practice to submit commented code.

Reply:
I kept the original given code in case anyone noticed errors with my code. It's easy for me to backtrack to see what I did wrong. But I can delete it.

Reply:
I pushed a new version of Datastore that doesn't include commented code.